CRM/Contribute/WorkflowMessage/RecurringEdit/BarbPending.ex.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class CRM_Contribute_WorkflowMessage_RecurringEdit_BarbPending extends \Civi\WorkflowMessage\WorkflowMessageExample {
+
+  public function getExamples(): iterable {
+    yield [
+      'name' => "workflow/{$this->wfName}/{$this->exName}",
```

Comment:
Non blocking - but when I had to debug this code I wound up re-writing this line because of the abbreviations - I switched to using a `getWorkflowName()` and `getExampleName` to reduce the cognitive load (I also re-wrote the class instantiation below and a renamed some other abbreviations in other files in order to trace through the code without my brain getting overloaded